roles/ceph-mds/tasks/containerized.yml

```diff
@@ -63,7 +63,7 @@
   changed_when: false
 
 - name: wait for mds socket to exist
-  command: "{{ docker_exec_cmd }} stat /var/run/ceph/{{ cluster }}-mds.{{ ansible_hostname }}.asok"
+  command: {{ docker_exec_cmd }} "stat /var/run/ceph/{{ cluster }}-mds.{{ ansible_hostname }}.asok || stat /var/run/ceph/{{ cluster }}-mds.{{ ansible_fqdn }}.asok"
```

Comment:
double quote the whole command

Reply:
I am using quotes to surround the docker exec command, we're executing docker exec only once